lib/locales/es.js

```diff
@@ -11,7 +11,64 @@
 module.exports = {
     moduleType: 'locale',
     name: 'es',
-    dictionary: {},
+    dictionary: {
+        'Autoscale': 'Autoescalar',                                   // components/modebar/buttons.js:139
+        'Box Select': 'Seleccionar Caja',                             // components/modebar/buttons.js:103
+        'Click to enter Colorscale title': 'Haga clic para introducir el título de la Escala de Color', // plots/plots.js:303
```

Comment:
hmm, these ones are getting a bit long, worries me about overflow. Do you think it would make sense to users that these are elements you can click and type new values if we leave off "Haga clic para" and just start at "Introducir" (or some conjugation thereof)?

Reply:
OK, it should be clear enough. In theory, [DRAE](http://dle.rae.es/?id=9SdJHbH) accepts `clicar`, but my experience is that `clicar` isn't so commonly used.